packages/backend/src/tools/JobQueue.ts

```diff
@@ -57,7 +59,7 @@ export class JobQueue {
       await job.execute()
       this.inProgress.splice(this.inProgress.indexOf(job), 1)
     } catch (error) {
-      this.logger.error(error)
+      this.logger.error(error, `Job "${job.name}" failed with:`)
```

Comment:
Maybe we can detect `arguments.length` and write `this.logger.error(message, error)`